src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Text } from '../typography/typography';
+
+const Checkbox = (props) => {
+  const [checked, setChecked] = useState(true);
+
+  const handleChange = () => {
+    setChecked(!checked);
+    props.onChange(checked);
```

Comment:
This will throw an exception if `props.onChange` is not defined. You can make this more failsafe by using ["optional chaining"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining):
```suggestion
    props.onChange?.(checked);
```